testing/stubs.go

```diff
@@ -109,6 +110,10 @@ func TestDecisionInterceptor(testID string, stubbedWorkflows, stubbedShortWorkfl
 					d.ScheduleActivityTaskDecisionAttributes.TaskList = &swf.TaskList{Name: S(*d.ScheduleActivityTaskDecisionAttributes.TaskList.Name + testID)}
 				}
 			}
+
+			if interceptor != nil {
+				interceptor.AfterDecision(decision, ctx, outcome)
```

Comment:
this needs to happen for all the functions in the interceptor (not just `AfterDecisionFn`)